packages/math-input/src/components/input/math-wrapper.ts

```diff
@@ -119,8 +119,8 @@ const KeysForJumpContext = {
 };
 
 class MathWrapper {
-    MQ: any;
-    mathField: any;
+    MQ: any; // MathQuill interface
+    mathField: any; // MathQuill input
```

Comment:
🔥 Nice!

Reply:
I tried to change the `mathField` name but it broke everything 😭